product/synquacer/module/synquacer_system/src/synquacer_main.c

```diff
@@ -253,6 +253,23 @@ static crg11_state_t crg11_state[CONFIG_SOC_CRG11_NUM] = {
     }
 };
 
+int synquacer_shutdown(void)
+{
+    volatile uint32_t *gpio = (uint32_t *)CONFIG_SOC_AP_GPIO_BASE;
+
+    /* set PD[9] high to turn off the ATX power supply */
+    gpio[5] |= 0x2; /* set GPIO direction to output */
+    __DSB();
+    synquacer_system_ctx.timer_api->delay(
```

Comment:
may be worth checking the returned status from this call to the timer api. Not sure if still recoverable, but perhaps a log may help should it fail.